cmd/minishift/cmd/start.go

```diff
@@ -231,6 +231,7 @@ func postClusterUp(machineName string, ip string, port int, routingSuffix string
 		atexit.Exit(1)
 	}
 
+	configurePeristentVolumes(sshCommander)
```

Comment:
Because of this the `minishift start` is taking more time now. In my test it is almost taking a minute more i.e. 4m24.133s (vs 3m38.676s  without this PR). Can we call this only when anyuid is enabled?